src/test/java/seedu/recipe/logic/commands/FindCommandTest.java

```diff
@@ -74,8 +75,10 @@ public void equals() {
     public void execute_namePropertyZeroKeywords_noRecipeFound() {
```

Comment:
Nitpicking, but `no` in English is followed by plural countable nouns, so `noRecipesFound`

Reply:
fixed